x/staging/posts/keeper/keeper.go

```diff
@@ -97,15 +94,17 @@ func (k Keeper) GetPost(ctx sdk.Context, id string) (post types.Post, found bool
 	return post, true
 }
 
-// GetPostChildrenIDs returns the IDs of all the children posts associated to the post
+// GetPostCommentsIDs returns the IDs of all the comments associated to the post
 // having the given postID
-// nolint: interfacer
-func (k Keeper) GetPostChildrenIDs(ctx sdk.Context, postID string) []string {
-	store := ctx.KVStore(k.storeKey)
+func (k Keeper) GetPostCommentsIDs(ctx sdk.Context, postID string) []string {
```

Comment:
I'm having a hard time understanding where this is used. Maybe we can delete it?

Reply:
I agree. It is only for test. Let me move it to test files.